lib/enkf/enkf_analysis.cpp

```diff
@@ -70,37 +70,37 @@ void enkf_analysis_fprintf_obs_summary(const obs_data_type * obs_data , const me
 
         for (int iobs = 0; iobs < obs_block_get_size( obs_block ); iobs++) {
           active_type active_mode = obs_block_iget_active_mode( obs_block , iobs );
-          if ((active_mode == MISSING) || (active_mode == LOCAL_INACTIVE))
-            continue;
+          const char * print_key;
+          if (iobs == 0)
+            print_key = obs_key;
+          else
+            print_key = "  ...";
+
+          fprintf(stream , obs_fmt , obs_count , print_key , obs_block_iget_value( obs_block , iobs ) , obs_block_iget_std( obs_block , iobs ));
+
+          if (active_mode == ACTIVE)
+            fprintf(stream , "  Active   |");
+          else if (active_mode == DEACTIVATED)
+            fprintf(stream , "  Inactive |");
+          else if (active_mode == LOCAL_INACTIVE)
+            fprintf(stream , "  Inactive |");
```

Comment:
Why are we printing the same status for both `DEACTIVATED` and `LOCAL_INACTIVE`? Think we could avoid some confusion on the future by differentiating in the log statement